routes/apiRoutes.js

```diff
@@ -0,0 +1,69 @@
+const apiRouter = require("express").Router();
+const path = require("path");
+const fs = require("fs");
+
+// uuid for unique id
+const { v4: uuidv4 } = require("uuid");
+const { request } = require("http");
+
+// API GET request
+apiRouter.get("/notes", (req, res) => {
+  // Read db.json file
+  let data = JSON.parse(
```

Comment:
update this to be a const please :-)
Variables inside of functions only 'exist' whilst the function is executing. Each time you call this method a new data variable will be created (and then destroyed once the function has executed).  Because of this, you should only use let if you are reassigning the value of a variable within the scope of the function.

Reply:
ok! :)